### ISSUE TITLE
Mapping headers

### ISSUE THREAD

**Issue by dclong** [harelba/q#145]
I think it's a good idea to create a mapping of headers (to unique valid field identifiers). The reason is that the header of a structured text is not always valid SQL identifiers. And I've seen duplicated field names in practice. Mapping the headers to unique valid SQL identifiers can help resolve the issues.

**Comment by harelba** [comment]
True. There are two parts to it actually: The first part is detecting problematic column names, which exists, but the duplicate-column check was implicit (I've just pushed a change to the master which makes it more explicit and managed). The second part is what you're referring to, allowing to provide the column names externally. This is indeed a good feature.
